planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp

```diff
@@ -188,13 +189,12 @@ rclcpp::Logger getLogger(const std::string & type);
  * The footprint is determined by the vehicle's pose and its dimensions, including the distance
  * from the base to the front and rear ends of the vehicle, as well as its width.
  *
- * @param ego_pose The current pose of the ego vehicle.
- * @param ego_info The structural information of the ego vehicle, such as its maximum longitudinal
- *                 offset, rear overhang, and width.
+ * @param common_data_ptr Shared pointer to CommonData that holds necessary ego vehicle's dimensions
+ *                        and pose information.
  *
  * @return Polygon2d A polygon representing the current 2D footprint of the ego vehicle.
  */
-Polygon2d getEgoCurrentFootprint(const Pose & ego_pose, const VehicleInfo & ego_info);
+Polygon2d get_ego_current_polygon(const CommonDataPtr & common_data_ptr);
```

Comment:
I think its more useful to keep it as
`Polygon2d get_ego_footprint(const Pose & ego_pose, const VehicleInfo & ego_info)`
so it can be used to get the footprint at any given pose not just current pose

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/9124/commits/176437636c61e6003af2d9b00f0d35e95afcdafb